src/components/dropdown-menu/dropdown-menu.tsx

```diff
@@ -14,16 +14,33 @@ export class DropdownMenu {
   @Prop() icon: boolean = false;
   @Prop() search: boolean = false;
   @Prop() filter: boolean = false;
+  @Event({ bubbles: false }) selectValues: EventEmitter<Object>;
```

Comment:
this event should propably bubble: true

Reply:
the bubbling is set to true by default, but I am making it false because I don't want the function to be invoked twice. Once triggered by the child-element, and second time by the parent wrapper.